automation/modules/backport_examples.py

```diff
@@ -49,6 +49,19 @@ def update_yaml_files(directory, version_to_replace, replacement):
                     content = f.read()
                 content = content.replace(version_to_replace, str(replacement))
                 print(version_to_replace, str(replacement))
+
+# String replacement in files for Kafka versions
+def replace_version_in_files(directory, version_to_replace, replacement, version_type):
+    for root, dirs, files in os.walk(directory):
+        for file in files:
+            if file.endswith(".yaml"):
+                file_path = os.path.join(root, file)
+                with open(file_path, 'r') as f:
+                    content = f.read()
+                if version_type == "kafka":
+                    content = content.replace(version_to_replace, str(replacement))
+                elif version_type == "inter_broker_protocol":
+                    content = content.replace(version_to_replace, str(replacement))
```

Comment:
Are these two conditionals really needed? It appears that regardless the value of `version_type` the replacement is going to take place for all content in the file. If that is the case, the `version_type` argument can be removed and
```
if version_type == "kafka":
  content = content.replace(version_to_replace, str(replacement))
elif version_type == "inter_broker_protocol":
  content = content.replace(version_to_replace, str(replacement))
```
can be replaced with just:
```
content = content.replace(version_to_replace, str(replacement))
```
Additionally, the `replace_version_in_files` only needs to be called once

One last note,  I would change the argument names:
```
`version_to_replace` -> `old_kafka_version`
`replacement` -> `new_kafka_version`
```